docs/shared/angular-tutorial/angular-monorepo.md

```diff
@@ -0,0 +1,1048 @@
+---
+title: 'Angular Monorepo Tutorial'
+description: In this tutorial you'll create a frontend-focused workspace with Nx.
+---
+
+# Building Angular Apps in an Nx Monorepo
+
+In this tutorial you'll learn how to use Angular with Nx in a [monorepo (integrated) setup](/concepts/integrated-vs-package-based#integrated-repos).
+
+What are you going to learn?
+
+- how to create a new Angular application
+- how to run a single task (i.e. serve your app) or run multiple tasks in parallel
+- how to leverage code generators to scaffold components
+- how to modularize your codebase and impose architectural constraints for better maintainability
+
+{% callout type="info" title="Looking for an Angular standalone app?" %}
+Note, this tutorial sets up a repo with applications and libraries in their own subfolders. If you are looking for an Angular standalone app setup then check out our [Angular standalone app tutorial](/getting-started/tutorials/angular-standalone-tutorial).
+{% /callout %}
+
+## Nx CLI vs. Angular CLI
+
+Nx evolved from being an extension of the Angular CLI to a [fully standalone CLI working with multiple frameworks](/getting-started/why-nx#how-does-nx-work). As a result, adopting Nx as an Angular user is relatively straightforward. Your existing code, including builders and schematics, will still work as before, but you'll also have access to all the benefits Nx offers.
+
+Advantages of Nx over the Angular CLI:
+
+- [Split a large angular.json into multiple project.json files](/concepts/more-concepts/nx-and-angular#projectjson-vs-angularjson)
```

Comment:
I feel like the advantages should point more towards the monorepo features given we have a monorepo tutorial. like the fact that with Nx you can create monorepos which you cannot really do with the Angular CLI (I mean, it works, but it's not made for that use case); scaling in terms of structuring (project.json files, libs etc), also the whole speed aspect like caching etc, ability to efficiently run multiple commands in a monorepo + CI scaling etc; and clearly modern tools which are integrated in a way to work efficiently in a monorepo setup which otherwise you need to figure out by yourself..